extensions/import/src/wizard/pages/summaryPage.ts

```diff
@@ -117,7 +117,19 @@ export class SummaryPage extends ImportPage {
 				newInPrimaryKey: val.primaryKey
 			};
 			const changeColumnResult = await this.provider.sendChangeColumnSettingsRequest(columnChangeParams);
-			changeColumnResults.push(changeColumnResult);
+			if (changeColumnResult.result.errorMessage) {
```

Comment:
you might want to check for `changeColumnResult` and `changeColumnResult.result` too or wrap this in a `try/catch`